benchmarks/hash_table/static_multimap/insert_bench.cu

```diff
@@ -60,41 +64,45 @@ std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> nvbench_static_multimap_i
 
                // Use timers to explicitly mark the target region
                timer.start();
-               map.insert(d_pairs.begin(), d_pairs.end(), launch.get_stream());
+               map.insert(pairs.begin(), pairs.end(), launch.get_stream());
                timer.stop();
              });
 }
 
-template <typename Key, typename Value, dist_type Dist, nvbench::int32_t Multiplicity>
+template <typename Key, typename Value, typename Dist>
 std::enable_if_t<(sizeof(Key) != sizeof(Value)), void> nvbench_static_multimap_insert(
-  nvbench::state& state,
-  nvbench::type_list<Key, Value, nvbench::enum_type<Dist>, nvbench::enum_type<Multiplicity>>)
+  nvbench::state& state, nvbench::type_list<Key, Value, Dist>)
 {
   state.skip("Key should be the same type as Value.");
 }
 
 using key_type   = nvbench::type_list<nvbench::int32_t, nvbench::int64_t>;
 using value_type = nvbench::type_list<nvbench::int32_t, nvbench::int64_t>;
-using d_type =
-  nvbench::enum_type_list<dist_type::GAUSSIAN, dist_type::GEOMETRIC, dist_type::UNIFORM>;
 
-using multiplicity = nvbench::enum_type_list<1, 2, 4, 8, 16, 32, 64, 128, 256>;
+NVBENCH_BENCH_TYPES(nvbench_static_multimap_insert,
+                    NVBENCH_TYPE_AXES(key_type, value_type, nvbench::type_list<dist_type::unique>))
+  .set_name("staic_multimap_insert_unique_occupancy")
+  .set_type_axes_names({"Key", "Value", "Distribution"})
+  .set_max_noise(3)                            // Custom noise: 3%. By default: 0.5%.
+  .add_int64_axis("NumInputs", {100'000'000})  // Total number of key/value pairs: 100'000'000
+  .add_float64_axis("Occupancy", nvbench::range(0.1, 0.9, 0.1));
```

Comment:
```suggestion
  .set_type_axes_names({"key", "value", "distribution"})
  .set_max_noise(3)                            // Custom noise: 3%. By default: 0.5%.
  .add_int64_axis("num_keys", {100'000'000})  // Total number of key/value pairs: 100'000'000
  .add_float64_axis("occupancy", nvbench::range(0.1, 0.9, 0.1));
```
Let's use snake case consistently for axis naming

Reply:
Good idea!

Reply:
nvbench uses camel-case for the other columns so the table header looks like this:
```
| key | value | distribution | num_keys  | occupancy | multiplicity | Samples | CPU Time  | Noise | GPU Time  | Noise | Elem/s | GlobalMem BW | BWUtil |
```

Reply:
Ah let's stay with camel-case then.